docs/digits_vae.md

```diff
@@ -0,0 +1,514 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.15.2
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
++++ {"id": "47OmRSTR1dJU"}
+
+# Debugging in JAX: a Variational autoencoder (VAE) model
+
+In [Getting started with JAX](https://jax-ai-stack.readthedocs.io/en/latest/getting_started_with_jax_for_AI.html) we built a simple neural network for classification of handwritten digits, and covered some of the key features of JAX, including its NumPy-style interface in the `jax.numpy`, as well as its transformations for JIT compilation with `jax.jit`, automatic vectorization with `jax.vmap`, and automatic differentiation with `jax.grad`.
+
+This tutorial will explore a slightly more involved model: a simplified version of a [Variational Autoencoder (VAE)](https://en.wikipedia.org/wiki/Variational_autoencoder) trained on the same simple digits data. Along the way, we'll learn a bit more about how JAX's JIT compilation actually works, and what this means for debugging JAX programs.
+
++++ {"id": "k19povzxp7hS"}
+
+## Loading the digits
+
+As before, we'll use the small, self-contained [scikit-learn digits dataset](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_digits.html) for ease of experimentation:
+
+```{code-cell}
+:id: aIwDAfS6PtFh
+:outputId: 4950f17a-7c47-4a83-cbcd-e206d07cca64
+
+from sklearn.datasets import load_digits
+from sklearn.model_selection import train_test_split
+import jax.numpy as jnp
+
+digits = load_digits()
+
+splits = train_test_split(digits.images, random_state=0)
+
+images_train, images_test = map(jnp.asarray, splits)
+
+print(f"{images_train.shape=}")
+print(f"{images_test.shape=}")
+```
+
++++ {"id": "2_Q16JRyrW7V"}
+
+The dataset comprises 1800 images, each represented by an 8x8 pixel grid.
+To see a visualization of this data, refer to [loading the data](https://jax-ai-stack.readthedocs.io/en/latest/getting_started_with_jax_for_AI.html#loading-the-data) in the previous tutorial.
```

Comment:
Unfortunately I can't add suggestions here, but you could use 
```
To see a visualization of this data, refer to [loading the data](getting_started_with_jax_for_AI.ipynb#loading-the-data) in the previous tutorial.
```
to create a link to the correct version of the target page (using hard links may cause the readthedocs page to be redirected to latest instead of stable and create unnecessary warnings).

Reply:
I prefer the full link so that it works when viewing the notebook directly (if we were just concerned about links in the rendered HTML, I'd use sphinx cross-references instead).